src/controllers/authenticate.controller.ts

```diff
@@ -9,6 +9,28 @@ class AuthenticateController {
   // Services
   private authenticateService = new AuthenticateService();
 
+  /**
+   * Retrive to fetch data of school and user on landing page
+   * - Called from client application after client signup
+   * - Http Call: Calls pathshala for fetching school details
+   * ```
+   * POST: /api/v1/platform/user
+   * ```
+   * @param req - The HTTP request object containing the user's login details.
+   * @param res - The HTTP response object used to send the response back to the client.
+   */
+  public initialUser = async (req: Request, res: Response) => {
+    const domain = req.get('host');
+    const schoolDetails = await this.authenticateService.initialUserRequest({ domain });
```

Comment:
is there a need to send it in object.
as ideally one controller should call one service only.
no other controller can call same service directly.
so ATM its fine to send it without object.